liquibase-jakarta-cdi/src/main/resources/schema.template.xml

```diff
@@ -0,0 +1,10 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ora="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
```

Comment:
Please update the dbchangelog xsd version to `dbchangelog-latest.xsd` unless there is a reason 2.0 is required.